test/e2e/storage/framework/testdriver.go

```diff
@@ -219,7 +219,7 @@ type DriverInfo struct {
 	// plugin if it exists and is empty if this DriverInfo represents a CSI
 	// Driver
 	InTreePluginName string
-	FeatureTag       string // FeatureTag for the driver
+	TestTags         []interface{} // tags for the driver (e.g. framework.WithSlow())
```

Comment:
This field cannot be set via YAML or JSON, which affects external users of the e2e.test binary - see PR description.